api/database/database.go

```diff
@@ -194,6 +195,11 @@ func MigrateDatabase(db *gorm.DB) error {
 		log.Printf("Failed to run exif fields migration: %v\n", err)
 	}
 
+  // Remove invalid GPS data from DB
```

Comment:
The indentation seems wrong. It should be tabs instead of spaces. Could you run `go fmt` here?